sdk/tests/js/glsl-conformance-test.js

```diff
@@ -132,24 +132,33 @@ function runOneTest(gl, info) {
   var vSource = info.vShaderPrep ? info.vShaderPrep(info.vShaderSource) :
     info.vShaderSource;
 
-  if (!quietMode())
+  if (!quietMode()) {
     wtu.addShaderSource(consoleDiv, vLabel, vSource);
+  }
 
   // Reuse identical shaders so we test shared shader.
   var vShader = vShaderDB[vSource];
   if (!vShader) {
-    vShader = wtu.loadShader(gl, vSource, gl.VERTEX_SHADER);
+    vShader = wtu.loadShader(gl, vSource, gl.VERTEX_SHADER, null, null, null, null, true);
+    let compiledVShader = vShader;
+    if (vShader && !gl.getShaderParameter(vShader, gl.COMPILE_STATUS)) {
+      compiledVShader = null;
+    }
     if (info.vShaderTest) {
-      if (!info.vShaderTest(vShader)) {
+      if (!info.vShaderTest(compiledVShader)) {
```

Comment:
What happens if a null shader is passed to vShaderTest (and fShaderTest, below)?